### DISCUSSION TITLE
Citekeys: how to use the Extra field IF there's an entry called "original-date", otherwise use regular year

### DISCUSSION THREAD

**Discussion by ReaderGuy42** [retorquere/zotero-better-bibtex#2622] [Q&A] [answered]
I'm working with lots of sources that get republished, but for quickly seeing when a source is from it would be useful if the citekey included the original publication year.

Is this possible somehow? That it would check if there is an entry called "original-date: " in the Extra field and then use that year, otherwise use the regular one?

Thanks :)

**Comment by retorquere** [comment, marked as answer]
```
( origyear || year )
```

**Reply by ReaderGuy42** [reply]
perfect, thank you!
